apps/i2c-accelerometer-gyroscope/src/i2c-accelerometer-gyroscope.c

```diff
@@ -0,0 +1,161 @@
+/* i2c-accelerometer-gyroscope.c - KNoT Application Client */
+
+/*
+ * Copyright (c) 2020, CESAR. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This example uses I2C communication protocol to get acceleration and angular
+ * velocity from a gy-251 sensor.
+ *
+ */
+
+#include <zephyr.h>
+#include <net/net_core.h>
+#include <logging/log.h>
+#include <device.h>
+#include <i2c.h>
+
+#include "knot.h"
+#include <knot/knot_types.h>
+#include <knot/knot_protocol.h>
+
+/* I2C slave address */
+#define MPU_ADDRESS			0X68
+
+/* MPU6050 internal registers */
+#define MPU6050_REG_PWR_MGMT1		0x6B
+#define MPU6050_SLEEP_EN		BIT(6)
+#define MPU6050_REG_GYRO_CFG		0x1B
+#define MPU6050_REG_ACCEL_CFG		0x1C
+#define MPU6050_REG_DATA_START		0x3B
+
+/* Full Scale Range to +- 2g */
+#define FS_SEL				0x00
+/* Full Scal Range to 250deg/seg */
+#define AFS_SEL				0x00
+
+/* Time interval (in mseg) to get new measures */
+#define TIME_INTERVAL			200
+
+/*
+ * Note that measured data has 16 bits, what makes necessary to put together
+ * the MSB's with their LSB's.
+ * Example:
+ * buf[0] will have MSB's for x axis acceleration and buf[1] will have LSB's
+ * for x axis acceleration, that is, the raw data of x axis acceleration will
+ * be buf16[0] = (buf[0] << 8) | buf(1)
+ */
+#define MERGE_MSB_LSB(a,b) ( a<<8 | b )
```

Comment:
Since it is an operator add spaces between it
```suggestion
#define MERGE_MSB_LSB(a,b) ( a << 8 | b )
```

Reply:
Done